### PR TITLE
Form page one complete

### PR DESCRIPTION
**Pull request by alan-fang**
We have completed the (elementary) frontend for 1.1 and 2.1.

### CODE REVIEW

File: frontend/src/components/FormOne.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+              items={[
+                { id: '1', value: 'Multiple Installments' },
+                { id: '2', value: 'Single End-Term Payments' }
+              ]}
+            />
+          </Form>
+
+          <br />
+          <Form inline>
+            <TextInput title="Loan Amount" info="ex. 5000" />
```
Comment (meghatron3000) [inline frontend/src/components/FormOne.js]: Make sure each input has a type validation.

File: frontend/src/components/FormOne.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            {'     '}
+
+            <TextInput title="Number of Terms" info="ex. 12" />
+          </Form>
+          <h2>
+            {' '}
+            <small> </small>{' '}
+          </h2>
```
Comment (meghatron3000) [inline frontend/src/components/FormOne.js]: why is there an empty header?

File: frontend/src/components/FormOne.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                { id: '1', value: 'Multiple Installments' },
+                { id: '2', value: 'Single End-Term Payments' }
+              ]}
+            />
+          </Form>
+
+          <br />
+          <Form inline>
+            <TextInput title="Loan Amount" info="ex. 5000" />
+            {'     '}
```
Comment (meghatron3000) [inline frontend/src/components/FormOne.js]: get rid of this

File: frontend/src/components/FormOne.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+                { id: '30', value: 'Per Month' },
+                { id: '90', value: 'Per Quarter' },
+                { id: '180', value: 'Per Half Year' },
+                { id: '365', value: 'Per Year' }
+              ]}
+            />
+          </Form>
+
+          <h2>
+            {' '}
```
Comment (meghatron3000) [inline frontend/src/components/FormOne.js]: get rid of these random spaces

File: frontend/src/components/LargeTextInput.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react'
+import './../styles/large-text-input.css'
+
+class LargeTextInput extends Component {
+  render() {
+    return (
+      <div className="large-text-input-container">
+        <span
```
Comment (meghatron3000) [inline frontend/src/components/LargeTextInput.js]: if your span is multiple lines please use <></>

File: frontend/src/components/TextInput.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+import './../styles/app.css'
+
+class TextInput extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      value: ''
+    }
+
+    this.handleInputChange = this.handleInputChange.bind(this)
```
Comment (meghatron3000) [inline frontend/src/components/TextInput.js]: use an anonymous function, don't manually bind this

File: frontend/src/components/TextInput.js [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+      <FormGroup controlId="formBasicText">
+        <ControlLabel>{this.props.title}</ControlLabel>
+        <FormControl
+          type="text"
+          value={this.state.value}
+          placeholder={this.props.info}
+          onChange={this.handleChange}
+          bsSize="small"
+        />
+        {'  '}
```
Comment (meghatron3000) [inline frontend/src/components/TextInput.js]: get rid of this

**Comment by alan-fang** [comment]
can u PLEASE merge this.?

**Comment by alan-fang** [comment]
can u PLEASE merge this..?

**Pull request merged**